app/src/test/java/org/oppia/android/app/player/state/StateFragmentLocalTest.kt

```diff
@@ -268,7 +272,7 @@ class StateFragmentLocalTest {
   }
 
   @Test
-  @Config(qualifiers = "port")
+  @Config(qualifiers = "+port")
```

Comment:
ah, we should always use `+`, right? Or is there a special case where we use `+` before port or land?
as this is new for me, so if you could help me understand better on this.

Reply:
I think `+port` means `"xlarge-port"`, is this correct?

If we are keeping these test only for a large screen, do we have a test for small screens?

Reply:
Using `+` overlays the qualifier onto any higher-level qualifiers, so this tests would actually be the same as the one specified in this test class, `port-xxhdpi`. The one below this would be `land-xxhdpi`. Does that help?

Reply:
So since we do specify a class qualifier I think we should always use `+`, otherwise I think it will override the class qualifiers we set to be default.